blenderproc/scripts/download_cc_textures.py

```diff
@@ -53,15 +56,17 @@ def cli():
                 break
         if do_not_use:
             continue
-        print("Download asset: {} of {}/{}".format(asset, index, len(data)))
         current_folder =  os.path.join(cc_texture_dir, asset)
         if not os.path.exists(current_folder):
             os.makedirs(current_folder)
-        current_file_path = os.path.join(current_folder, "{}.zip".format(asset))
+        elif os.listdir(current_folder):
```

Comment:
That does not ensure that all files for a certain material are correctly unpacked. :(

Reply:
As far as I understand it, everything is unpacked at the end with `SetupUtility.extract_from_response(current_folder, response)` which, if fails, will raise according to `SetupUtility.extract_file()`:
- `IOError` that should only happen when you have no permissions to write, so you didn't even create the main output_folder where you save all textures
- `zipfile.BadZipfile` which I think makes some kind of checksum before unpacking and therefore if raised, no documents are extracted
In both cases, no files are added to the folder and it is safe to just resume the downloads. My experience is that it is always the cc_textures server that closes the connection and we do not obtain a file and then download cc_textures fails.

Maybe there is some other kind of error that can happen when unpacking compressed files that I am not aware of and then we need as you suggest a way to check that output is correct, or in case of error, just delete the current folder before exiting. Any suggestions?